Makefile

```
@@ -15,10 +18,14 @@ build:
 debug:
 	go run -race $(LDFLAGS) .
 
+.PHONY: lint
+lint:
+	golangci-lint run ./... --timeout 3m --tests=false --skip-dirs=features
```

most apps use the below, is there a reason for the (extra) flags?
```suggestion
	golangci-lint run ./...
```